<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LrcPage</class>
 <widget class="QWidget" name="LrcPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1030</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="bgStyle" native="true">
     <property name="styleSheet">
      <string notr="true">#bgStyle{
	background:#462940;
}
*{
 color : #FFFFFF;
}</string>
     </property>
     <widget class="QWidget" name="songsBox" native="true">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>0</y>
        <width>600</width>
        <height>610</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <widget class="QWidget" name="titleBox" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>600</width>
         <height>80</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <widget class="QLabel" name="musicName">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>5</y>
          <width>500</width>
          <height>40</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="musicSinger">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>50</y>
          <width>500</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="lrcContent" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>90</y>
         <width>600</width>
         <height>520</height>
        </rect>
       </property>
       <widget class="QLabel" name="line1">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>10</y>
          <width>530</width>
          <height>50</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>600</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="line2">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>70</y>
          <width>530</width>
          <height>50</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>600</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="line3">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>130</y>
          <width>530</width>
          <height>50</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>600</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="line4">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>280</y>
          <width>530</width>
          <height>50</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>600</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="line5">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>340</y>
          <width>530</width>
          <height>50</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>600</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="line6">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>400</y>
          <width>530</width>
          <height>50</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>600</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="lineCenter">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>190</y>
          <width>530</width>
          <height>80</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>600</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">#lineCenter
{
	color:#1ECE9A;
}</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="hideBtn">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#hideBtn
{
 border:none;
}</string>
      </property>
      <property name="text">
       <string>OPEN</string>
      </property>
     </widget>
     <widget class="QLabel" name="imageBox">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>130</y>
        <width>331</width>
        <height>351</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
